### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
##Mantis / PMDB link:
PMDB: https://pmdb.fortinet.com/ProjectManagement/editprojects.php?id=16498
Mantis:https://mantis.fortinet.com/bug_view_page.php?bug_id=0926818

##Description of changes:
Changed websocket subscribe from 'Module' to a 'Particular Record'

##Diff in the PR reviewed?
Yes

##Base branch correct?
Yes

##Unit tests performed and results:
- Installation Verified
- Widget listens to an update in the record
- Widget does not listen to any other change in the module(CRUD)
- Edit view verified
- Documentation Update

### CODE REVIEW

File: widget/info.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "title": "SLA Count Down Timer",
+  "version": "2.0.1",
+  "published_date":"1601528772",	
+  "releaseNotes": "available",
+  "metadata": {
+   "pages": [
+    "View Panel"
+   ],
+   "snapshots": [
+    "https://repo.fortisoar.fortinet.com/fsr-widgets/slaCountDownClock-1.0.2/images/sla-countdown-timer-settings.png",
```
Comment (cs-pradeep-gurav) [inline widget/info.json]: "version": "2.0.1",

Comment (cs-yash-patodkar) [reply widget/info.json]: Done

File: widget/view.controller.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                var days = Math.floor(total / (1000 * 60 * 60 * 24));
+                return {
+                    total,
+                    days,
+                    hours,
+                    minutes,
+                    seconds
+                };
+            }
+            else {
```
Comment (cs-pranali-chavan) [inline widget/view.controller.js]: Initialize these parameters once and add values in if block and return them once.

**Review by cs-pranali-chavan — CHANGES_REQUESTED** [review]
Comment added.

File: widget/edit.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
+<form data-ng-submit="save()" name="editCountDownClock">
+    <div class="modal-header">
+        <h3 class="modal-title col-md-9">Edit SLA CountDown Timer</h3>
+        <button type="button" class="close" data-ng-click="cancel()" data-dismiss="modal" aria-label="Close"
+            id="close-edit-relationship-subtab-btn">
```
Comment (cs-pradeep-gurav) [inline widget/edit.html]: close-edit-count-down-clock-btn

**Review by cs-pradeep-gurav — APPROVED** [review]
Looks good.

Address the minor comment and do the sanity from dev content hub portal.

**Review by cs-pranali-chavan — APPROVED** [review]
Looks good

**Pull request merged**